### PR TITLE
Add quota stat

### PR DESCRIPTION
**Pull request by sam-e-ch**
This PR add the "Quota" stat, which tells you how long you still have to work to fill your quota in the month up to this day.

### CODE REVIEW

File: lib/stats.rb [hunk 1]
```diff
@@ -185,6 +186,13 @@ def eight_am(day)
     Time.new(day.year, day.month, day.day, 8)
   end
 
+  def quota
+    workdays_passed = workdays.count - workdays_left
+    target = ((config.daily_goal * workdays_passed).round(2) * 3600).floor
+
+    Totals.format((target - monthly_total))
```
Comment (rathrio) [inline lib/stats.rb]: Double parens probably obsolete.

File: lib/stats.rb [hunk 2]
```diff
@@ -185,6 +186,13 @@ def eight_am(day)
     Time.new(day.year, day.month, day.day, 8)
   end
 
+  def quota
```
Comment (rathrio) [inline lib/stats.rb:189]: Would you mind adding a simple test to `test/stats_test.rb` that describes an expected result here?

**Review by rathrio — COMMENTED** [review]
Neat ❤️

LGTM besides comments.

**Comment by sam-e-ch** [comment]
@rathrio I added a test

**Pull request merged**